rio_tiler_pds/__init__.py

```diff
@@ -1,13 +1,6 @@
 """rio-tiler-pds."""
 
-from functools import lru_cache
-
-from rio_tiler.utils import aws_get_object
+from . import cbers, landsat, modis, sentinel  # noqa
+from .utils import get_object  # noqa
```

Comment:
I moved `get_object` into a submodule!

Reply:
Do you want to include `get_object` as a public export? This allows `from rio_tiler_pds import get_object`...

Seems not ideal... is there existing code that imports `get_object` globally?